cengage/react-magma
# docs(select): Reorganize docs

_Issue: none_

## What I did

- Updated Select docs to be in a more coherent order (basic props at the top, all customizations together, etc.)
- Updated Multi Select placeholder spacing
- Updated Input doc examples to use the InputType enum (instead of a string)

## Screenshots (if appropriate):
**before**
![image](https://user-images.githubusercontent.com/91160746/228357256-051b0fe2-5300-49a1-91e6-11821475e769.png)

**after**
![image](https://user-images.githubusercontent.com/91160746/228357594-46b17309-0c76-4f38-9c31-832a50249993.png)


## Checklist 

## How to test

- Navigate to the Select docs site and make sure the content flows better
- Confirm that the spacing is updated for multi selects
